cluster/cluster.go

```diff
@@ -1048,14 +1048,17 @@ func (c *Cluster) PrePullK8sImages(ctx context.Context) error {
 	var errgrp errgroup.Group
 	hostList := hosts.GetUniqueHostList(c.EtcdHosts, c.ControlPlaneHosts, c.WorkerHosts)
 	hostsQueue := util.GetObjectQueue(hostList)
-	for w := 0; w < WorkerThreads; w++ {
+	imageList := []string{c.SystemImages.Kubernetes, c.Services.Kubelet.InfraContainerImage}
+	for w := 0; w < len(imageList); w++ {
```

Comment:
We still need to loop over `WorkerThreads` cuz that controls the concurrency for the hostsQueue below. With this change now we'd be only creating 2 goroutines instead of worker threads count which will impact performance for bigger clusters. The change you did at L1057 will suffice for pre-pulling sandbox image, right?